packages/strapi-tiptap-editor/admin/src/translations/nl.json

```
@@ -1,6 +1,8 @@
 {
   "common.words.freeProduct": "gratis",
   "common.action.cancel": "Annuleren",
+  "common.action.insert": "Invoegen",
+  "common.action.preview": "Voorbeeldweergave",
```

Het is net wat anders... maar wat denk je hier van?

```suggestion
  "common.action.preview": "Bekijk voorbeeld",
```